src/OpenTelemetry/Internal/BatchExportActivityProcessorOptions.cs

```diff
@@ -21,7 +21,7 @@
 
 namespace OpenTelemetry.Trace
 {
-    public class BatchExportActivityProcessorOptions : BatchExportProcessorOptions<Activity>
+    internal sealed class BatchExportActivityProcessorOptions : BatchExportProcessorOptions<Activity>
```

Comment:
Given this class is implementing something as per sec, I think its safe to expose this as public. 
The alternate of keeping internal and using internalvisible might cause other issues  like exporters accidently accessing something it was not meant to, or hide issues affecting other exporter which don't get the same privilege as the ones in this repo.

Reply:
@CodeBlanch what do you think here?
(Internal would be fine, if we can use the "compile incude" option, rather than specially exposing internals to set of exporters..)

Reply:
I tried a couple of things. Ultimately pushed a change to fix the benchmark project by using an alias. It is nasty, but at least the nasty is localized to just benchmarks. Check it out, LMK. Might be worth just making it public 🤣 

One neat thing I found doing this, you can link in an entire folder like this:

```xml
  <ItemGroup>
    <Compile Include="$(RepoRoot)\src\OpenTelemetry.Api\Internal\**\*.cs" LinkBase="Includes\" />
  </ItemGroup>
```

So we could possibly clean some of this up (somewhat) by organizing things into a couple of standard folders we link into exporters, instrumentation, etc?

Reply:
Sorry that it took me so long, but I was 🤒 

> Ultimately pushed a change to fix the benchmark project by using an alias

Thank you 👍 

> Might be worth just making it public

I think it makes sense if we do not have any concrete argument against doing it (other than that it will increase the API). Do we have an example that making it public would make some harm?

> you can link in an entire folder like this

I think it can be good. However, probably we do not want to include the whole `Internal` folder but rather some subfolder(s). Also, I think it could be done in a separate PR.

Reply:
We discussed this in todays SIG meeting. Cijo/Alan agree to make this public, as it is implementing something from the spec.